src/lib/PreviewUI.js

```diff
@@ -285,6 +289,28 @@ class PreviewUI {
         this.progressBar.finish();
     }
 
+    /**
+     * Shows a notification message.
+     *
+     * @private
```

Comment:
Should be public, as it's being called in Preview.js (and needs to be called by anything else using it)

Ditto to `hideNotification()`